lightly/active_learning/scorers/classification.py

```diff
@@ -156,7 +171,10 @@ def _get_scores_uncertainty_least_confidence(self):
         return scores, "uncertainty_least_confidence"
 
     def _get_scores_uncertainty_margin(self):
-        scores = 1 - _margin_largest_secondlargest(self.model_output)
+        if self.model_output.shape[1] == 1:
+            scores = np.zeros_like(self.model_output)
+        else:
+            scores = 1 - _margin_largest_secondlargest(self.model_output)
         return scores, "uncertainty_margin"
```

Comment:
In the first case, the scores will have shape `n x 1` and in the second case only `n` right? I would make this consistent

Reply:
Did it